scripts/testnet/sovereignBridge/pyScripts/genesis_contract.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+import shutil
+import sys
+
+
+def copy_wasm_in_project(file_path: str, wasm_path: str):
+    try:
+        if os.path.exists(file_path):
+            os.remove(file_path)
+
+        shutil.copy2(os.path.expanduser(wasm_path), file_path)
+        print(f"{wasm_path} copied successfully.")
+    except FileNotFoundError:
+        print(f"File {wasm_path} not found.")
+    except PermissionError:
+        print("Permission denied.")
+    except Exception as e:
+        print(f"An error occurred: {e}")
+
+
+def push_genesis_contract(file_path: str, genesis_contract):
```

Comment:
Is this name ok? what is the genesis contract?

Reply:
It's a genesis contracts "push" in the genesis contracts list